apps/public-docsite-v9/src/AccessibilityScenarios/TabListOverflow.stories.tsx

```diff
@@ -70,8 +70,6 @@ const tabs: SettingsTab[] = [
 
 type OverflowMenuItemProps = {
   tab: SettingsTab;
-  // FIXME: This should be a consistent callback type
-  // eslint-disable-next-line @nx/workspace-consistent-callback-type
```

Comment:
going forward stories are not checked against our internal react rules ( only production code ) - that's why this lint disable is now unused -> I removed it

Reply:
Just curious: why are stories not checked? Since they are used for documentation shouldn't we hold them to the same standards as the source code?

Reply:
right, so it was turned off before for public-docsite already:

<img width="1928" alt="image" src="https://github.com/user-attachments/assets/87d182e6-1d0c-4eb7-8c9f-64f7e2f3da63">

setup is driven by following config:

<img width="2065" alt="image" src="https://github.com/user-attachments/assets/5cd100a5-b6de-4382-bd36-4924c5d63de3">


I agree that future-wise, no-globals should be enabled in story files that are part of official docs in order to "lead by example" for our users  
- should be handled via separate initiative, instead of going eslint-disable route
- this would need additional changes to lint setup ( as for example we dont wanna enable that callback rule for stories - that's purely internal implementation detail about how to define callbacks )


thoughts ?